web/src/tippyjs.js

```diff
@@ -29,6 +29,38 @@ function get_tooltip_content(reference) {
     return "";
 }
 
+// We need to store all message list instances together to destroy them in case of re-rendering.
+const message_list_tippy_instances = new Set();
```

Comment:
How does this work as a `Set`? Can you explain why Tippy instances, which are complex objects, will be hashed/stored in a way that causes requesting deletion to actually work here?

Reply:
I guess it's the same object, not a copy or anything, so this should be OK.